### PR TITLE
Explainer updates for clarity, grammar, and URL localization

### PR DESCRIPTION
**Pull request by alexandrascript**
This PR fixes a spelling error, suggests changes for clarity in certain parts of the explainer, and removes English/US-based URL structure from links so that readers are directed to a localized URL.

### CODE REVIEW

**Comment by domenic** [comment]
The spelling changes and URLs are reasonable, but please don't insert your personal style choices here (e.g. abbreviations vs. not).

**Comment by KenjiBaheux** [comment]
I caught an extra one: "support for combining the langauge detection" => language.

**Comment by alexandrascript** [comment]
@domenic I've reverted some of the suggestions, which reflect Google's style guide, but I know don't have to apply here.

However, I've kept the changes to the paragraph on line 380. The original paragraph was missing clarity--some is easy to define, such as the pronouns. A less prescriptive example is the change to "which did"--this sentence fragment would be better understood as "with" because it's reflective of what's being "included" in the earlier version.

File: README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ Browsers are increasingly offering language translation to their users. Such tra
...
 
-An important supplement to translation is language detection. This can be combined with translation, e.g. taking user input in an unknown language and translating it to a specific target language. In a similar way, browsers today often already have langauge detection capabilities, and we want to offer them to web developers through a JavaScript API.
+An important supplement to translation is language detection. This can be combined with translation, e.g. taking user input in an unknown language and translating it to a specific target language. In a similar way, browsers today often already have language detection capabilities, and we want to offer them to web developers through a JavaScript API.
 
 ## Goals
 
 Our goals are to:
 
-* Help web developers perform real-time translations (e.g. of user input).
+* Help web developers perform real-time translations (e.g., of user input).
```
Comment (domenic) [inline README.md:20]: Please revert

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ The following are explicit non-goals:
 
 The following are potential goals we are not yet certain of:
 
-* Allow web developers to know whether translations are done on-device or using cloud services. This would allow them to guarantee that any user data they feed into this API does not leave the device, which can be important for privacy purposes. (Similarly, we might want to allow developers to request on-device-only translation, in case a browser offers both varieties.)
+* Allow web developers to know whether translation is done on-device or using cloud services. This would allow them to guarantee that any user data they feed into this API does not leave the device, which can be important for privacy purposes. (Similarly, we might want to allow developers to request on-device-only translation, in case a browser offers both varieties.)
```
Comment (domenic) [inline README.md:34]: Please revert

File: README.md [hunk 3]
```diff
@@ -377,7 +377,7 @@ Another possible simplification is to make the `capabilities()` APIs synchronous
 
 ### Allowing unknown source languages for translation
 
-An earlier revision of this API including support for combining the langauge detection and translation steps into a single translation call, which did a best-guess on the source language. The idea was that this would possibly be more efficient than requiring the web developer to do two separate calls, and it could possibly even be done using a single model.
+An earlier version of this API included support for combining the langauge detection and translation steps into a single translation call, with a best-guess on the source language. The idea implied that this approach would be more efficient than requiring two separate calls, and that translation could possibly be done using a single model.
```
Comment (domenic) [inline README.md:380]: Please revert

**Review by domenic — CHANGES_REQUESTED** [review]

**Review by Thapeloza — APPROVED** [review]
```
```
- - [ ] 
```
```

**Pull request closed without merging**